modules/mlvis-common/package.json

```diff
@@ -10,7 +10,7 @@
     "Lezhi Li <lezhi.li@uber.com>",
     "Yang Wang <gnavvy@uber.com>"
   ],
-  "main": "dist/index.js",
+  "main": "dist/esm/index.js",
```

Comment:
Looks like we're doing this somewhat differently from deck.gl : https://github.com/uber/deck.gl/blob/2732d1192132eaf0a2784116b7a0b69fb2143706/modules/mapbox/package.json#L19-L21

Just want to make sure this is intentional and an informed decision.

Reply:
Yeah, the plan is to only support esm build for now.